tests/sdk/v2/data/sample_project/workflow_defs_no_raise.py

```diff
@@ -0,0 +1,20 @@
+################################################################################
+# © Copyright 2022 Zapata Computing Inc.
+################################################################################
+import helpers  # type: ignore
+
+import orquestra.sdk as sdk
+
+
+@sdk.task()
+def capitalize_task(text):
+    return helpers.capitalize_helper(text)
+
+
+@sdk.workflow(default_source_import=sdk.InlineImport())
+def sample_workflow():
+    return [capitalize_task("sample")]
+
+
+if __name__ == "__main__":
+    model = sample_workflow.model
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**nitpick:** ‏I think this is left over debug code?

Reply:
No - it is intentionally here to ensure that mechanism in unit test to launch this script as main works properly